linux_os/guide/system/software/sudo/package_sudo_installed/rule.yml

```diff
@@ -22,6 +22,7 @@ identifiers:
 
 references:
     nist: AC-6(5)
+    srg: SRG-OS-000480-GPOS-00227,SRG-OS-000378-GPOS-00163
```

Comment:
nack, sudo does not authenticate peripherals like USB drives


SRG-OS-000378-GPOS-00163 |   | The operating system must authenticate peripherals before establishing a connection.
-- | -- | --

Reply:
fixed.